packages/Badge/src/Badge.story.tsx

```diff
@@ -2,22 +2,14 @@ import React from 'react';
 import { storiesOf } from '@storybook/react';
 import { text, select } from '@storybook/addon-knobs';
 import Badge from './Badge';
+import { Variant } from '.';
 
 storiesOf('Badge', module).add('Default', () => (
   <Badge
     variant={select(
       'Variant',
-      [
-        'default',
-        'danger',
-        'warning',
-        'darkBlue',
-        'lightBlue',
-        'primary',
-        'outline',
-        'dark',
-      ],
-      'darkBlue',
+      Object.values(Variant) as Array<Variant>,
```

Comment:
Interesting... Does TS not infer `as Array<Variant>`?

Reply:
Nope, explanation [here](https://github.com/microsoft/TypeScript/pull/12253#issuecomment-263132208) for why `Object.keys`, `Object.values`, and `Object.entries` doesn't infer quite as much as you might think. Although it'd be nice if they could make an exception for enums...

Reply:
Shame, thanks for the explanation!